src/components/Admin/ManageQuestions.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component, Fragment } from "react";
+import styled from "styled-components";
+
+import Button from "react-bootstrap/Button";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+import Toast from "react-bootstrap/Toast";
+import AdminLayout from "./AdminLayout";
+import QuestionDisplay from "./QuestionDisplay";
+import AddQuestion from "./AddQuestion";
+
+import { AuthUserContext, withFirebase } from "upe-react-components";
+
+import Loader from "../Loader";
+import { Container } from "../../styles/global";
+
+const StyledDiv = styled.div`
+  text-align: right;
+`;
+
+class ManagerQuestions extends Component {
+  constructor(props) {
+    super(props);
+
+    this.updatePage = this.updatePage.bind(this);
+    this.toggleAdd = this.toggleAdd.bind(this);
+  }
+
+  _initFirebase = false;
+  state = {
+    questionList: null,
+    settings: null,
+    loading: true,
+    error: null,
+    addQuestion: false,
+  };
+  static contextType = AuthUserContext;
+  unsub = null;
+
+  componentDidMount() {
+    if (this.props.firebase && !this._initFirebase) this.loadSettings();
+  }
+
+  componentDidUpdate(prevProps) {
+    if (this.props.firebase && !this._initFirebase) this.loadSettings();
+  }
+
+  componentWillUnmount() {
+    if (typeof this.unsub === "function") this.unsub();
+  }
+
+  updatePage = () => {
+    this.setState({ addQuestion: false });
+    this.loadSettings();
+  };
+
+  loadSettings = async () => {
+    this._initFirebase = true;
+
+    this.props.firebase
+      .questions()
+      .get()
+      .then((querySnapshot) => {
+        const questionList = querySnapshot.docs.map((doc) => {
+          return { id: doc.id, ...doc.data() };
+        });
+        this.setState({ questionList, loading: false }, () => {
+          console.log("Questions Loaded");
+        });
+      });
+  };
+
+  toggleAdd = () => {
+    this.setState({ addQuestion: !this.state.addQuestion });
+  };
+
+  render() {
+    const { loading, error, questionList, addQuestion } = this.state;
+
+    if (loading) return <Loader />;
+    if (error)
+      return (
+        <Container flexdirection="column">
+          <h1>{error}</h1>
+        </Container>
+      );
+
+    const authUser = this.context;
```

Comment:
authUser isn't being used here, should be removed

Reply:
Resolved in commit bb4137083f7a98fa345509e8c14a6aa6b59d602a